Babylon/commands/api/scenarios/get.py

```diff
@@ -0,0 +1,49 @@
+from logging import getLogger
+from typing import Any
+
+from click import command, argument, option
+
+from Babylon.commands.api.scenarios.service.api import ScenarioService
+from Babylon.utils.credentials import pass_azure_token
+from Babylon.utils.decorators import timing_decorator, wrapcontext, retrieve_state
+from Babylon.utils.response import CommandResponse
+
+logger = getLogger("Babylon")
+
+
+@command()
+@wrapcontext()
+@pass_azure_token("csm_api")
+@timing_decorator
+@retrieve_state
+@option("--organization_id", "organization_id", type=str)
+@option("--workspace_id", "workspace_id", type=str)
+@argument("scenario_id", required=False)
+def get(
+    state: Any,
+    organization_id: str,
+    workspace_id: str,
+    scenario_id: str,
+    azure_token: str,
+) -> CommandResponse:
+    """
+    Get scenario details
+    """
+    if not scenario_id:
+        logger.error("Scenario id is missing")
+        return CommandResponse.fail()
+    service_state = state["state"]
+    if organization_id:
+        service_state["api"]["organization_id"] = organization_id
+    if workspace_id:
+        service_state["api"]["workspace_id"] = workspace_id
+
+    # change this line when scenario manipulation will be specified
+    spec = {"scenario_id": scenario_id}
+
+    scenario_service = ScenarioService(state=service_state, azure_token=azure_token, spec=spec)
```

Comment:
scenario_service could be renamed to service here and in other files, same for service_state that could be just state